activerecord/test/cases/connection_pool_test.rb

```diff
@@ -491,19 +500,16 @@ def test_anonymous_class_exception
         end
       end
 
-      class ConnectionTestModel < ActiveRecord::Base
-        self.abstract_class = true
-      end
```

Comment:
This test class has been split into two, one for thread tests and one for fiber, because there were some issues with running both test suites twice against the same class, some connection state that wasn't being totally cleaned up that I couldn't nail down. The failures weren't specific to the new Fiber code; simply running the threads test suite twice in a row produced the issue.